src/plugins/map/MapObject.cpp

```diff
@@ -1070,7 +1079,17 @@ MapObjectMapLoc::MapObjectMapLoc(const MapLocParams& params, const std::string&
 {
 	GenerateLabel();
 
-	UpdateFromParams(params);
+	m_mapLocParams = params;
+	Update(true);
+}
+
+MapObjectMapLoc::MapObjectMapLoc()
+{
+	m_isCreatedFromDefaultLoc = false;
+	m_initialized = false;
+	m_index = -1;
+	m_tag = "";
+	m_labelText = "";
```

Comment:
strings default initialize to empty, don't need this.

Reply:
actually you're already initializing all of these in the header, don't need any of them.